### DISCUSSION TITLE
Python in Homebrew vs OS

### DISCUSSION THREAD

**Discussion by mattpackwood** [Homebrew/discussions#1740] [Linux]
Hi, I am upgrading some of my "fleet" from Ubuntu 20.10 to 21.04 so I am now getting Python 3.9 in the OS, is there a way to use the version of Python that "comes with the OS" rather than have a "duplicate Python install"?

**Comment by SMillerDev** [comment]
If you mean "use it in homebrew" then the answer is no. We don't support using system components instead of formulae because it's much harder to determine which version the system part is (and if it's compatible)
